src/constants/index.ts

```diff
@@ -335,3 +335,8 @@ export const TYPE_AND_SWAP_NOT_SUPPORTED_CHAINS: ChainId[] = [ChainId.ZKSYNC, Ch
 export const SWAP_FEE_RECEIVER_ADDRESS = '0x4f82e73EDb06d29Ff62C91EC8f5Ff06571bdeb29'
 
 export const TOKEN_SCORE_TTL = 86400
+
+export const ROUTES_API_PATHS = {
```

Comment:
Just suggesting better name IMO
```suggestion
export const AGGREGATOR_API_PATHS = {
```